app/components/client_upgrade_listener/client_upgrade_listener.js

```diff
@@ -55,24 +56,30 @@ export default class ClientUpgradeListener extends PureComponent {
         };
     }
 
+    static getDerivedStateFromProps(nextProps, prevState) {
```

Comment:
Similarly, you also need to return the new value of `isLandscape` from this function whenever it changes so that this will work correctly if the device ever rotates again.

Reply:
actually `isLandscape` is not needed in the state, in fact you could just listen for `Dimensions` changes and apply the new top value there.

**Note:** There is no animation going on here as far as I can see, only applying a new top value. Perhaps is best to forget about this file and create a separate ticket for this one.